components/concurrency_manager/src/lib.rs

```diff
@@ -105,13 +105,15 @@ pub struct ConcurrencyManager {
 }
 
 impl ConcurrencyManager {
+    // This method should be used in test only. To create a new concurrency manager,
+    // please use `new_with_config` instead.
     pub fn new(latest_ts: TimeStamp) -> Self {
         Self::new_with_config(
             latest_ts,
             DEFAULT_LIMIT_VALID_DURATION,
             ActionOnInvalidMaxTs::Panic,
             None,
-            Duration::ZERO,
+            DEFAULT_LIMIT_VALID_DURATION + Duration::from_secs(1),
```

Comment:
We need to update the unit tests accordingly.
Alternatively, we could make this method UT-specific and create a separate one for GC (as a temporary solution)

Reply:
How about:
- rename `ConcurrencyManager::new` to sth like `ConcurrencyManager::new_dummy` and make it `#[cfg(test)]`
- let `GcRunnerCore::new_txn` use `ConcurrencyManager::new_with_config` directly (or "create a separate one for GC", eg `ConcurrencyManager::new_for_gc`).

Reply:
Yes `#[cfg(test)]` is a cleaner way but introduces many changes, I'll try it another time.

Reply:
@ekexium @zyguan 
Refactored the `new` to `new_for_test` and a new `new_for_gc` is created as a temporary solution, PTAL